docs/wiki/The CI environment - Static validation.md

```diff
@@ -54,6 +54,10 @@ In particular, each resource's API version is compared with those currently avai
 
 This test also leverages the [`utilities/pipelines/staticValidation/module.tests.ps1`](https://github.com/Azure/ResourceModules/blob/main/utilities/pipelines/staticValidation/module.tests.ps1) script.
 
+To test the API versions, the test leverages the file `utilities/src/apiSpecsList.json` file as a reference for all API versions available for any used Provider Namespace & Resource Type.
```

Comment:
Going forward, we'll need to document this more in detail, e.g., how is apiSpecsList.json created and maintained. We can do that in a separated PR, but we need to create an issue for it.

Reply:
added a bit more detail